### PR TITLE
feat: swap link-token with cw20-base

### PR DESCRIPTION
**Pull request by denismaxim0v**

### CODE REVIEW

File: contracts/aggregator-proxy/src/contract.rs [hunk 1]
```diff
@@ -324,10 +325,21 @@ mod tests {
 
     pub fn instantiate_link(app: &mut App) -> Addr {
         let link_id = app.store_code(contract_link_token());
+        let main_balance = Cw20Coin {
+            address: Addr::unchecked("owner").into(),
+            amount: Uint128::from(1_000_000_000 as u128),
```
Comment (mradkov) [inline contracts/aggregator-proxy/src/contract.rs:330]: Is this value supposed to be hardcoded here? Wouldn't that result in minting `1_000_000_000` at the time of instantiation?

File: contracts/flux-aggregator/src/integration_tests.rs [hunk 2]
```diff
@@ -64,12 +64,24 @@ fn default_init() -> (App, Addr, Addr, Addr) {
     let mut router = mock_app();
     let owner = Addr::unchecked("owner");
 
+    let main_balance = Cw20Coin {
+        address: owner.clone().into(),
+        amount: Uint128::from(1_000_000_000 as u128),
```
Comment (mradkov) [inline contracts/flux-aggregator/src/integration_tests.rs:69]: Same as above?

File: contracts/aggregator-proxy/src/contract.rs (hunk 1 above)
Comment (denismaxim0v) [reply contracts/aggregator-proxy/src/contract.rs:330]: does it matter in the tests?
With the real instantiation we pass that in the InstantiateMsg

Comment (mradkov) [reply contracts/aggregator-proxy/src/contract.rs:330]: Oops, my bad. :D I did not notice this was part of the tests.

**Review by mradkov — APPROVED** [review]

**Review by nzaharov — COMMENTED** [review]
You could remove the old link_token artifact

**Comment by mradkov** [comment]
@denismaxim0v is this swap going to affect something in the demo setup [here](https://github.com/hackbg/terra-chainlink-demo)? Should some artifacts be swapped there as well or sth else?

**Pull request merged**